test/helper.rb

```diff
@@ -3,8 +3,8 @@
 require "bundler/setup"
 Bundler.require(:default, :test)
 
-require "minitest/pride"
 require "maxitest/autorun"
```

Comment:
Moving `maxitest/autorun` above the other minitest related requires caused "line focusing" to start working again.